setup_builder.py

```diff
@@ -155,7 +161,9 @@ def _get_packages() -> Iterator[str]:
                 if "__init__.py" in os.listdir(directory):
                     yield directory
 
-        pkgs = list(_get_packages())
+        pkgs = self._bsec.packages()
```

Comment:
Further down there's a check `if len(pkgs) > 1: raise Exception`. Do we want to support mutl-packages? If so, we'll have to think about the other uses of `FromFiles.package`.

Reply:
Note, multi-package semantic versioning is supported: https://github.com/WIPACrepo/mou-dashboard/blob/master/setup.cfg#L7

Reply:
I left that as-is, for a further PR to decide.